modules/olm-installing-opm.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-managing-custom-catalogs.adoc
+
+[id="olm-installing-opm_{context}"]
+= Installing `opm`
+
+To install the `opm` CLI tool on your workstation:
+
+.Prerequisites
+
+* `podman` version 1.4.4+
+
+.Procedure
+
+ifdef::openshift-enterprise,openshift-webscale[]
+. Set the `REG_CREDS` environment variable to the file path of your registry
+credentials for use in later steps. For example, for the `podman` CLI:
++
+----
+$ REG_CREDS=${XDG_RUNTIME_DIR}/containers/auth.json
+----
+
+. Authenticate with `registry.redhat.io`:
++
+----
+$ podman login registry.redhat.io
+----
+endif::[]
+
+. Extract the `opm` binary from the Operator Registry image and copy it to your
+local file system:
++
+ifdef::openshift-origin[]
+----
+$ oc image extract quay.io/openshift/origin-operator-registry:4.5.0 \
+    --path /usr/bin/opm:. \
+    --confirm
+----
+endif::[]
+ifdef::openshift-enterprise,openshift-webscale[]
+----
+$ oc image extract registry.redhat.io/openshift4/ose-operator-registry:v4.4 \
```

Comment:
It appeared registry.redhat.io/openshift4/ose-operator-registry:v4.4 at http://file.rdu.redhat.com/~adellape/060120/new_olm/operators/olm-managing-custom-catalogs.html#olm-mirror-catalog-image_olm-managing-custom-catalogs , I'm not sure if that is expected, but just a warning

Reply:
Whoops, this was meant to be 4.5, yeah.